src/fsharp/absil/ilwrite.fs

```diff
@@ -2219,9 +2223,13 @@ let GetFieldDefTypeAsBlobIdx cenv env ty =
                                               EmitType cenv env bb ty)
     GetBytesAsBlobIdx cenv bytes
 
-let GenPdbImport (cenv: cenv) env (input: ILDebugImport) =
+let GenPdbImport (cenv: cenv) _env (input: ILDebugImport) =
```

Comment:
I believe this is only used one other place so the _env can probably just be removed.